yaksh/forms.py

```diff
@@ -274,6 +274,17 @@ def __init__(self, *args, **kwargs):
 class CourseForm(forms.ModelForm):
     """ course form for moderators """
 
+    def save(self, commit=True, *args, **kwargs):
```

Comment:
I have to side with @ankitjavalkar  on this. We might need this feature in the future.